index.js

```diff
@@ -1,7 +1,20 @@
+#!/usr/bin/env node
+
 const fs = require('fs')
 const sax = require('sax')
 const json2xml = require('jsontoxml')
 const filePath = process.argv[2]
+const outputPath = process.argv[3]
+
+if (!filePath) {
+  console.error('First argument must be the path to an (X)HTML file')
+  process.exit(111)
+}
+
+if (!outputPath) {
+  console.error('Second argument must be the path to an output (X)HTML file')
```

Comment:
Instead of an error if not provided, might it be helpful to write to stdout instead?
```javascript
if (!outputPath) {
  process.stdout.write(outputXml)
} else {
  fs.writeFileSync(outputPath, outputXml)
}
```